modules/tools/00_mod.mk

```diff
@@ -109,6 +109,10 @@ KUBEBUILDER_ASSETS_VERSION=1.28.3
 TOOLS += etcd=$(KUBEBUILDER_ASSETS_VERSION)
 TOOLS += kube-apiserver=$(KUBEBUILDER_ASSETS_VERSION)
 
+# Additional tools can be defined to reuse the tooling in this file
+ADDITIONAL_TOOLS ?=
+TOOLS += $(ADDITIONAL_TOOLS)
```

Comment:
Another option is to use `TOOLS ?= ` here and in the other place that adds a tool.